app/javascript/components/landing-page.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+<div>
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 2 spaces but found 0 spaces  vue/html-indent